lib/theme_check/language_server/handler.rb

```diff
@@ -71,6 +81,23 @@ def on_text_document_completion(id, params)
 
       private
 
+      def in_memory_storage(root)
+        config = ThemeCheck::Config.from_path(root)
+
+        # Make a real FS to get the files from the snippets folder
+        fs = ThemeCheck::FileSystemStorage.new(
+          config.root,
+          ignored_patterns: config.ignored_patterns
+        )
+
+        # Turn that into a hash of empty buffers
+        files = fs.files
+          .map { |fn| [fn, ""] }
```

Comment:
Just curious why you need to do this?

Reply:
Because I need a list of snippets from the RenderSnippetCompletionProvider. The InMemoryStorage doesn't have anything in it and I didn't want to have both an InMemoryStorage instance for buffers and a FSStorage for files _just_ so that the render snippet completion provider was able to list those files.

Reply:
Oh I see! Because that's what `InMemoryStorage.new` expects! 👍